PrintInformationPatches.py

```diff
@@ -177,33 +178,44 @@ def _formatdAffixes(self) -> None:
             profile_name = self._global_stack.qualityChanges.getName()
         material_name = "%s %s" % (extruder_stack.material.getMetaDataEntry("brand"), extruder_stack.material.getName())
 
+        pre_sliced = "?" if self._print_information._pre_sliced else ""
```

Comment:
A `?` as part of the filename may be illegal in certain OSes, as it is commonly used as a wildcard character. It may also result in issues where the jobname is used in an (improperly escaped) URL.

Reply:
It appears that the filename being send is on a best effort, but Octoprint sanitised the file to its own discretion to consider it safe.

But I understand your point.